python/lsst/sims/utils/healpyUtils.py

```diff
@@ -172,3 +172,55 @@ def healbin(ra, dec, values, nside=128, reduceFunc=np.mean, dtype=float):
     """
     return _healbin(np.radians(ra), np.radians(dec), values, nside=nside,
                     reduceFunc=reduceFunc, dtype=dtype)
+
+
+def moc2array(data, uniq, nside=128, reduceFunc=np.sum, density=True, fillVal=0.):
+    """Convert a Multi-Order Coverage Map to a single nside HEALPix array. Useful
+    for converting maps output by LIGO alerts. Expect that future versions of
+    healpy or astropy will be able to replace this functionality. Note that this is
+    a convienence function that will probably degrade portions of the MOC that are
+    sampled at high resolution.
+
+    Parameters
+    ----------
+    data : np.array
+        Data values for the MOC map
+    uniq : np.array
+        The UNIQ values for the MOC map
+    nside : int (128)
+        The output map nside
+    reduceFunc : function (np.sum)
+        The function to use to combine data into single healpixels.
+    density : bool (True)
+        If True, multiplies data values by pixel area before applying reduceFunc, and divides
+        the final array by the output pixel area. Should be True if working on a probability density MOC.
+    fillVal : float (0.)
+        Value to fill empty HEALPixels with. Good choices include 0 (default), hp.UNSEEN, and np.nan.
+    """
```

Comment:
Can you include a reference to the MOC specification somewhere in this docstring.  It looks like you are using a bunch of terms of art that are MOC-specific.  It would be nice if users could see where those are defined.